ops/pebble.py

```diff
@@ -1136,6 +1137,12 @@ def _request_raw(
         if query:
             url = url + '?' + urllib.parse.urlencode(query)
 
+        # python 3.5 urllib requests require their data to be a bytes object -
+        # generators won't work.
+        if sys.version_info[1] <= 5:
```

Comment:
Would `sys.version_info[:2] <= (3, 5)` or even `sys.version_info < (3, 6)` be a better way of describing this?

Reply:
Also, what about `if A and B: ...` rather than the nested `if`?

Reply:
I'll merge the ifs.  But operator framework doesn't even work on or support python2 so the major version check seems a bit meh.

Reply:
I suppose my check wouldn't work well once we get to python 4...

Reply:
Yeah, I was thinking of code clarity and a potential version 4.x :-)